tests/Microsoft.Bot.Builder.AI.QnA.Tests/QnAMakerCardEqualityComparer.cs

```diff
@@ -39,7 +45,7 @@ public bool Equals(IActivity x, IActivity y)
                 }
             }
 
-            return true;
+            return string.Equals(activity1.Text, activity2.Text);
```

Comment:
I don't think this overall method works correctly. IT's been a while since I wrote an Comparer, but I belive you're missing things.

Reply:
With introduction of new display activity for supporting Precise Answer feature, this change was needed. Ensured that  the test coverage was good and old tests passed.

Reply:
This comparer is in use for simple card comparison in QnAMaker tests.

---
In reply to: [430563747](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430563747) [](ancestors = 430563747)